config/locales/devise.he.yml

```diff
@@ -0,0 +1,61 @@
+---
+he:
+  devise:
+    confirmations:
+      confirmed: '&rlm;כתובת הדוא"ל אומתה בהצלחה.'
+      send_instructions: '&rlm;נשלח אליך דוא"ל עם הוראות לאימות כתובת הדוא"ל שאמור להתקבל בדקות הקרובות.'
+      send_paranoid_instructions: '&rlm;אם כתובת הדוא"ל שלך קיימת במסד הנתונים, יתקבל בדקות הקרובות דוא"ל עם הוראות לאימות כתובתך.'
+    failure:
+      already_authenticated: &rlm;חשבון זה כבר מחובר.
```

Comment:
btw, some had %{references} in the string, so I used double quotes, as I've not checked what library is parsing the strings. If you prefer single quotes everywhere, I'll fix that.